.github/workflows/grabthecam-build.yml

```diff
@@ -70,3 +69,88 @@ jobs:
         cd build
         cmake ..
         make
+
+  test-build:
```

Comment:
Having `test-build` and `test` in their current form feel quite redundant. We checkout sources, install dependencies and proceed with the rest. The whole checkout + dependencies take ~1 minute, while the rest of the steps take couple of seconds.

I guess we can either:

* have `test-build` which will run its steps + `pyvidctrl` check so that all things requiring vivid will happen in the same job
* have one `test` job running those two jobs

Reply:
Good point, I think that having one job, is going to be the cleanest approach